### ISSUE TITLE
no-duplicate-tags does not work at examples level

### ISSUE THREAD

**Issue by xgbuils** [vsiakka/gherkin-lint#169]
``` gherkin
Feature: Feature with duplicate tags

Scenario Outline: This is a Scenario Outline with two duplicate example tags
  Then this is a then step <foo>
@sametag @sametag
Examples:
  | foo |
  | bar |
```
does not report an error.
